controllers/birdsController.js

```diff
@@ -0,0 +1,79 @@
+//DB
+const db = require("../models");
+
+//GET /api/birds -- get all birds
+function index(req, res) {
+  db.Bird.find({}, function(err, allBirds) {
+    if (err) {
+      console.log("Failed /api/birds ");
+      return;
+    }
+    res.json(allBirds);
+  })
+}
+
+//POST /api/birds -- adding a bird
+function create(req, res) {
+  db.Bird.create(req.body, function(err, bird) {
+    console.log('entering db.Bird.create()');
+    if (err) {
+      console.log(`create bird failed: err =${err}`);
+      return;
+    }
+    console.log("create bird success");
+    res.json(bird);
+  });
+}
+
+function update(req, res) {
+  db.Bird.findById(req.params.birdId, function(err, bird){
+    if (err) {
+      console.log(`Did not find bird id: ${req.params.birdId} in db`);
+      return;
```

Comment:
You should probably send a response here, not just return from the function; as is, your app.js file would be waiting forever, for a response that would never come, if there were an error. Instead, you could send a 404.

Reply:
And this goes for all of your error-handling: you should send a response, not just console.log.